guides/common/modules/proc_creating-a-custom-certificate.adoc

```diff
@@ -75,19 +75,21 @@ authorityKeyIdentifier=keyid,issuer
 [ alt_names ]
 DNS.1 = _{context}.example.com_ <3>
 ----
-<1> In the `[ req_distinguished_name ]` section, enter information about your organization.
 ifeval::["{context}" == "{project-context}"]
+<1> In the `[ req_distinguished_name ]` section, enter information about your organization.
 <2> Set the certificate's Common Name `CN` to match the fully qualified domain name (FQDN) of your {ProductName}.
 To confirm a FQDN, on that {ProductName}, enter the `hostname -f` command.
 This is required to ensure that the `katello-certs-check` command validates the certificate correctly.
+<3> Set the Subject Alternative Name (SAN) `DNS.1` to match the fully qualified domain name (FQDN) of your server.
 endif::[]
 ifeval::["{context}" == "{smart-proxy-context}"]
+<1> In the `[ req_distinguished_name ]` section, enter information about your organization.
 <2> Set the certificate's Common Name `CN` to match the fully qualified domain name (FQDN) of your {ProductName} or a wildcard value `*`.
 To confirm a FQDN, on that {ProductName}, enter the `hostname -f` command.
 This is required to ensure that the `katello-certs-check` command validates the certificate correctly.
 If you set a wildcard value, you must add the `-t {certs-proxy-context}` option when you use the `katello-certs-check` command.
-endif::[]
 <3> Set the Subject Alternative Name (SAN) `DNS.1` to match the fully qualified domain name (FQDN) of your server.
+endif::[]
```

Comment:
This, for some reason, throws an error. When all numbers are together in if-block it works. I could not figure out what context value is causing that, so I put them together.